rust/protocol/src/storage/inmem.rs

```diff
@@ -1,38 +1,52 @@
 //
-// Copyright 2020 Signal Messenger, LLC.
+// Copyright 2020-2022 Signal Messenger, LLC.
 // SPDX-License-Identifier: AGPL-3.0-only
 //
 
+//! Implementations for stores defined in [super::traits].
+//!
+//! These implementations are purely in-memory, and therefore most likely useful for testing
+//! right now.
+//!
+//! *TODO: Define a serialization scheme for all of the in-memory stores so that they can be
+//! persisted to file or elsewhere!*
```

Comment:
Eh. *Someone* can do this, but I don't know if it will be a goal for this library. It's more likely that the library will start including a SQL-based implementation that's separate from the in-memory stores.

Reply:
Oops! That was a personal note for my own use case. Removed, but thank you for sketching that possible implementation!